packages/shared/src/lib/core/wallet/actions/isAddressATrackedToken.ts

```diff
@@ -0,0 +1,7 @@
+import { NetworkId } from '@core/network/types'
+import { IProfile } from '@core/profile'
+
+export function isAddressATrackedToken(networkId: NetworkId, address: string, profile: IProfile): boolean {
```

Comment:
Maybe this sounds better?
```suggestion
export function isTrackedTokenAddress(networkId: NetworkId, address: string, profile: IProfile): boolean {
```